.travis.yml

```diff
@@ -8,6 +8,8 @@ go:
   - 1.4.x
```

Comment:
@ameliagapin You can drop 1.2 - 1.7. We only need/want to CI the most recent two major Go releases

Reply:
Got it.

Updated.